app/code/Magento/InventoryInStorePickup/Model/Carrier/Command/GetShippingPrice.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryInStorePickup\Model\Carrier\Command;
+
+use Magento\InventoryInStorePickupApi\Model\Carrier\GetShippingPriceInterface;
+use Magento\Quote\Model\Quote\Address\RateRequest;
```

Comment:
One more evidence that we need a separate module. I;d suggest not to depend on quote here.

Reply:
Moved to separated module.